storage/remote/write.go

```diff
@@ -65,13 +68,15 @@ func NewWriteStorage(logger log.Logger, reg prometheus.Registerer, walDir string
 		logger = log.NewNopLogger()
 	}
 	rws := &WriteStorage{
-		queues:        make(map[string]*QueueManager),
-		reg:           reg,
```

Comment:
Once `NewQueueManager` doesn't need the `reg` arg anymore, this can go from the struct, too.